packages/mobile/src/screens/sign-on-screen/screens/SelectArtistScreen/SelectArtistsScreen.tsx

```diff
@@ -0,0 +1,107 @@
+import { useCallback } from 'react'
+
+import {
+  selectArtstsPageMessages as messages,
+  selectArtistsSchema
+} from '@audius/common'
+import { addFollowArtists } from 'common/store/pages/signon/actions'
+import { getGenres } from 'common/store/pages/signon/selectors'
+import { Formik } from 'formik'
+import { createMaterialCollapsibleTopTabNavigator } from 'react-native-collapsible-tab-view'
+import { useDispatch, useSelector } from 'react-redux'
+import { toFormikValidationSchema } from 'zod-formik-adapter'
+
+import { Flex, Text } from '@audius/harmony-native'
+
+import { ReadOnlyAccountHeader } from '../../components/AccountHeader'
+import { Heading, PageFooter } from '../../components/layout'
+
+import { SelectedGenresTabBar } from './SelectedGenresTabBar'
+import { TopArtistsCardList } from './TopArtistsCardList'
+
+const Tab = createMaterialCollapsibleTopTabNavigator()
+
+type SelectArtistsValues = {
+  selectedArtists: string[]
+}
+
+const initialValues: SelectArtistsValues = {
+  selectedArtists: []
+}
+
+export const SelectArtistsScreen = () => {
+  const genres = useSelector((state: any) => [
+    'Featured',
+    ...(getGenres(state) ?? [])
+  ])
+  const dispatch = useDispatch()
+
+  const renderHeader = useCallback(
+    () => (
+      <Flex pointerEvents='none' backgroundColor='white'>
+        <ReadOnlyAccountHeader />
+        <Flex>
+          <Heading
+            heading={messages.header}
+            description={messages.description}
+            gap='s'
+            ph='l'
+            pv='unit10'
+            pb='s'
+          />
+        </Flex>
+      </Flex>
+    ),
+    []
+  )
```

Comment:
A bit weird looking like this, this could just be a standalone component `Header` above

Reply:
yeah im down to refactor that, the idea was that it's cause the component takes "renderHeader" but can prob just do `renderHeader = () => <Header />`